2_process/src/process_precip.R

```diff
@@ -190,6 +194,11 @@ process_precip_rasters <- function(ind_file, precip_spatial_ind,
 
   parallel::stopCluster(cl)
 
+  # Name the rasters the same as the timestep they represent
+  #   8.23.2021: this part used to just happen and downstream
+  #   steps depend on the naming to be like this.
+  names(rasters) <- as.character(time$time)
```

Comment:
So this makes it seem like the variable in this object is `time` but line 188 above makes it seem like it is `date_time`. I would expect them to use the same one?